chains/avalanche-testnet.json

```diff
@@ -4,7 +4,12 @@
   "id": "43113",
   "symbol": "testAVAX",
   "testnet": true,
-  "providerUrl": "https://api.avax-test.network/ext/bc/C/rpc",
+  "providers": [
+    {
+      "alias": "avax-test",
```

Comment:
I just used the domain (without TLD) as the alias. Let me know if you want any of these changed

Reply:
I think we should consider using `default` and `public` wherever applicable as per https://github.com/wagmi-dev/viem/blob/5c95fafceffe7f399b5b5ee32119e2d78a0c8acd/src/types/chain.ts#L57

Reply:
Ah yeah - I like that. Do you have a preference for making the existing ones `default` or `public`?

Reply:
(I went with `default` - let me know if you disagree)

Reply:
Yup, I had the same preference [here](https://github.com/api3dao/chains/pull/85#discussion_r1376084871)